en/04_Changelogs/5.1.0.md

```diff
@@ -48,6 +49,16 @@ Note that even if you store the schema in the cache directory, the schema still
 
 See [choosing where the cache is stored](/developer_guides/graphql/getting_started/building_the_schema/#cache-location) for more information about configuring the schema storage directory.
 
+### CMS page search performance
+
+- The CMS search has been optimised to reduce the number of database queries made when searching for pages when the site uses elemental content blocks. This has resulted in a small performance improvement.
```

Comment:
Can we give a generic baseline here? 

Something like "In our test environment, with X number of pages we observed a reduction of Y% number of queries and of z% response time. Performance will vary with your environment."

Reply:
Updated